mtp/server.go

```diff
@@ -138,6 +143,24 @@ type InfoPayload struct {
 	Frame  []byte   `json:"frame"`
 }
 
+func (s *LVServer) AFFocusNow() {
+	log.LV.Info("AFFocusNow: focus now")
+	select {
+	case s.afNowChan <- true:
+	default:
+	}
+}
+
+func (s *LVServer) AFFocusAfter(after time.Duration) {
+	go func() {
+		select {
+		case <-time.After(after):
+			s.AFFocusNow()
+		case <-s.ctx.Done():
+		}
+	}()
+}
+
```

Comment:
1. AFFocusNow is used only once. Let's merge it into the caller.
2. The entire contents of AFFocusAfter is wrapped by a `go func() {}()`. In this case, creating the new goroutine on the caller's side is cleaner and provides more choice in terms of concurrency (sync/async call etc.)

```suggestion
func (s *LVServer) AFFocusAfter(after time.Duration) {
	select {
	case <-time.After(after):
	case <-s.ctx.Done():
		return
	}

	select {
	case s.afNowChan <- true:
	case <-s.ctx.Done():
		return
	}
}

```

Reply:
done